.github/actions/dbtest/action.yaml

```diff
@@ -9,7 +9,7 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - uses: actions/setup-go@v4
+    - uses: actions/setup-go@v5
```

Comment:
Please check all actions other than `setup-go`, too.
Some actions are not updated even though new ones are being released.

Reply:
I find Actions in whole repository and if find it, changed it.